.circleci/test.sh

```diff
@@ -6,13 +6,13 @@ set +o pipefail
 
 ROOT=$(dirname $0)/..
 EXIT_STATE=0
-MAX_AUTO_RETRY=5
 
 log () {
     echo -e "\n$1"
 }
 
 # inspired by https://unix.stackexchange.com/a/82602
+MAX_AUTO_RETRY=1
```

Comment:
Why would we want to lower `MAX_AUTO_RETRY`?

Reply:
It looks flaky tests (jasmine3) sometime required more retries. The gl tests on the other hand are slow and may not need that number of retries. Specially if they fail for a good reason, we don't want to wait too long to be notified that the test is actually failed.

Reply:
> The gl tests on the other hand are slow and may not need that number of retries.

Well, I suspect the gl tests are slow (now) because you reduced the number of test per shards to 1.

To me, having `MAX_AUTO_RETRY=5` for all retry loops is the best of both world. Sufficient retry attempt, but failing runs that take to long to exit.